app.js

```diff
@@ -3,13 +3,64 @@ const Promise = require('bluebird');
 const notifier = require('node-notifier');
 const spotify = require('./lib/spotify.js');
 const objectPath = require('object-path');
+const log = require('./lib/log.js');
 
 const CsvReader = require('promised-csv');
 const reader = new CsvReader();
 
-const DEFAULT_START_VOLUME = 25;
+const commandLineUsage = require('command-line-usage');
+const commandLineArgs = require('command-line-args');
+
+const optionDefinitions = [{  name: 'duration',  alias: 'd',  type: Number,  defaultValue: 10,
+  typeLabel: '{underline seconds}',
+  description: 'The total duration of the alarm.',
+}, {  name: 'volume',  alias: 'v',  type: Number,  typeLabel: '{underline percent}',
+  description: 'The volume of the alarm on Spotify.',
+}, {  name: 'fade-in',  alias: 'i',  type: Number,  typeLabel: '[{underline seconds}]',
+  description: 'Whether or not to fade in, optionally the time to fade in.',
+}, {  name: 'fade-out',  alias: 'o',  type: Number,  typeLabel: '[{underline seconds}]',
+  description: 'Whether or not to fade out, optionally the time to fade out.',
+}, {  name: 'help',  alias: 'h',  type: Boolean,  description: 'This help dialog',
```

Comment:
Did you mean "*The* help dialog"?

Reply:
I meant "This" because it will be the dialog currently being viewed.